lib/src/components/NxTextInput/NxTextInput.tsx

```diff
@@ -87,7 +87,7 @@ export const PrivateNxTextInput = forwardRef<HTMLDivElement, Props>(
             * presentational concern
             */}
           {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions*/}
-          <div className="nx-text-input__box" onClick={setFocusToInput}>
+          <div className="nx-text-input__box custom" onClick={setFocusToInput}>
```

Comment:
What's this?

Reply:
Sorry, that was when I was testing something and meant to be removed. Thanks for catching it

Reply:
removed in 87b9ab3